engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/new/views/new-repository.html

```diff
@@ -103,6 +103,15 @@ <h4 translate> Sync Settings </h4>
           Comma separated list of processor architectures to filter the sync by. Example: amd64
         </p>
       </div>
+      <div ng-show="repository.content_type === 'deb'" bst-form-group label="{{ 'Errata URL' | translate }}">
+        <input id="deb_errata_url"
+               name="deb_errata_url"
+               ng-model="repository.deb_errata_url"
+               type="text"/>
+        <p class="help-block" translate>
+          URL to a deb_errata service. Example: https://dep.atix.de/dep/api/v1/debian
```

Comment:
deb_errata seems out of place in this context. How about "URL to a Debian errata service" ?

Reply:
Agreed

Reply:
In that case, shouldn't it be "URL to an APT errata service" (to not discriminate against Ubuntu)?
"deb" works for both, but "Debian" really doesn't...

Reply:
Ah good point. I'd be happy with "URL to a deb errata service" which is basically what was there in the first place but without the '_' 

I'm not sure if APT is the best terminology since we're not thinking about the package manager here